syntax = "proto3";


enum ServeChannel {
    ALIPAY = 0;
    WECHAT_PAY = 1;
}

enum PayMethod {
    // 支付宝当面付
    ALIPAY_F2F = 0;
}

message Trade {
    // trade_id: 传给支付方的ID
    string id = 1;
    ServeChannel channel = 2;
    PayMethod pay_method = 3;
    int64 create_time = 4;
    int64 update_time = 5;
    // 第三方订单信息
    string trade_info = 6;
    int32 pay_fee = 7;
    // 业务服务的订单ID(单笔订单多次支付的情况下，该ID可能会重复)
    string order_id = 8;
}
